app/forms/form.rb

```diff
@@ -43,7 +43,9 @@ def form_attributes(attribute_names)
       attributes = (application_attributes + member_attributes + navigator_attributes)
       self.attribute_names = attributes
 
-      attribute_strings = attribute_names.map(&:to_s)
```

Comment:
This adds support for `member_attributes(foo: [])` rather than just `member_attributes(:foo)`